.github/workflows/tests.yaml

```diff
@@ -423,15 +436,14 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install bun
-        run: npm install -g bun
-
       - uses: pnpm/action-setup@v2
         with:
           version: latest
 
-      - name: Install bun
-        run: npm install -g bun
```

Comment:
I'm not sure if this is a duplicate or necessary.

Reply:
Probably duplicate.